semanticdb/metacp/src/main/scala/scala/meta/internal/javacp/Javacp.scala

```diff
@@ -402,7 +411,10 @@ object Javacp {
   private def sarray(tpe: s.Type): s.Type =
     styperef("_root_.scala.Array#", tpe :: Nil)
 
-  private def styperef(symbol: String, args: List[s.Type] = Nil, prefix: Option[s.Type] = None): s.Type = {
+  private def styperef(
+      symbol: String,
+      args: List[s.Type] = Nil,
+      prefix: Option[s.Type] = None): s.Type = {
```

Comment:
Why wasn't this autoformatted before?

Reply:
the `.java` part of `project.excludeFilters` ignored everything in the `javacp` directory. Fixed in https://github.com/scalameta/scalameta/pull/1417

Note to self, I should make this configuration easier in scalafmt. Or alternatively, we should let scalafmt loose on the scalameta repo! 😄